### DISCUSSION TITLE
When will the structure modules be available?

### DISCUSSION THREAD

**Discussion by alejocrojo09** [facebookresearch/esm#314] [General]
Dear all,

I want to congratulate you on the massive work on ESMFold and its release with the protein language models. However, I was wondering if you have an expected time on when the folding trunk modules will be available. 

Thank you very much!

**Comment by gieses** [comment]
I am just watching the repo (no affiliation with ESM), but please check:
1) https://github.com/facebookresearch/esm/issues/311
2) https://github.com/facebookresearch/esm/issues/269
